Week3/homework/step2-6.js

```diff
@@ -2,15 +2,19 @@
 
 const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
-
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const arr2 = arr.reduce(function(newArr, arrItem) {
```

Comment:
Great use of `reduce`. You could even `return` directly on line 6 - instead of assigning it to a new variable which you return later (`arr2`)